### ISSUE TITLE
anomalous-backslash-in-string for raw+binary string literal

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#272]
Originally reported by: **Anthony Sottile (BitBucket: [asottile](http://bitbucket.org/asottile), GitHub: @asottile?)**

---

```
$ cat test.py
CONST = br'\.'
$ pylint test.py
************* Module test
W:  1, 0: Anomalous backslash in string: '\.'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
$ pylint --version
pylint 1.2.1, 
astroid 1.1.1, common 0.62.0
Python 2.7.3 (default, Feb 27 2014, 19:58:35) 
[GCC 4.6.3]
```

Expected: no warning

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/272

**Comment by rogalski** [comment]
Repository top already dropped support for Python 2.6.

It can be either solved by `import lib2to3.pgen2.tokenize as tokenize` when `sys.version_info < (2,7,4)` or closed as 'will-not-fix'.

**Comment by PCManticore** [comment]
Let's close this. While using ``lib2to3`` could solve this problem for Python 2.7, I think doing this right now would be superfluous. Python older than 2.7.4 is not really a priority for us (and dropping support for Python 2 altogether in one-two years isn't that far-fetched). Another reason for this is that the tests seem to fail with this change on Python 3 and I don't want to spend energy on a minor issue.

**Comment by asottile** [comment]
Can confirm this works in 2.7.12 (xenial) -- probably not worth fixing
